app/liquid/views/partials/_form.liquid

```diff
@@ -14,6 +14,8 @@
           <input type="email" name="{{field.name}}" class="form-control mailcheck form__content" id="" placeholder="{{ field.label }}" />
         {% when 'phone' %}
           <input type="tel" name="{{field.name}}" class="form-control form__content" id="" placeholder="{{ field.label }}" />
+        {% when 'postal' %}
+          <input type="text" name="{{field.name}}" class="form-control form__content" id="" placeholder="{{ field.label }}" />
```

Comment:
this looks like it's the same as the `text` case, right? so it can just be another option in that `when` statement